data/rulei/rulei ships.txt

```diff
@@ -23,8 +23,6 @@ ship "vyu-Ira"
 		"cost" 77777777
 		"shields" 68000
 		"hull" 49000
-		"required crew" 1	
-		"bunks" 100
```

Comment:
dont you need to add automaton for it to behave properly?
if you tested this fine thuogh